README.md

```diff
@@ -70,42 +61,98 @@ AirbrakesV2/
 ├── README.md
 ```
 
+## Local Setup
+
+**This project uses Python 3.12. Using an older version may not work since we use newer language features**
+
+### Clone the repository:
+
+```
+git clone https://github.com/NCSU-High-Powered-Rocketry-Club/AirbrakesV2.git
+cd AirbrakesV2
+```
+
+### Set up a virtual environment:
+
+```bash
+python -m venv .venv
+
+# For Linux
+source .venv/bin/activate
+# For Windows
+venv\Scripts\activate
+```
+
+### Install the required dependencies:
+
+```bash
+pip install .[dev]
+```
+_There are libraries that only fully work when running on the Pi (gpiozero, mscl), so if you're having trouble importing them locally, program the best you can and test your changes on the pi._
```

Comment:
we should mention `mscl` installation instructions.

Reply:
I was thinking about that, but do you think it's necessary? It's already installed/setup on the pi, and having it installed on your laptop doesn't matter that much

Reply:
It's important for the future people who want to setup `mscl` on their Pi's, e.g. I had to do it when swapping the SD card